### PR TITLE
Intercom v2 source updates

### PR DESCRIPTION
**Pull request by ameypv-rudder**
## What do these changes do?

> Updated Intercom v2 source documentation
> Updated Intercom v2 URL
> Removed older Intercom version from the sidebar

## Doc preview

https://rudderstack-docs-hsf9m0b5f-rudder-stack.vercel.app/sources/extract/intercom/

## Type of documentation

- [ ] New documentation
- [x] Documentation update
- [ ] Hotfix

### CODE REVIEW

File: docs/sources/extract/intercom.mdx [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+
+### Selecting the data to import
+
+You can choose the Intercom data that you want to ingest by selecting the required resources.
+
+<img src="../../assets/cloud-extract-sources/intercomv2-5.png" alt="Selecting the data to import" />
+
+The below table mentions the sync types supported by the Intercom resources while syncing data to your warehouse destination:
+
+| Resource | Full Refresh sync | Incremental sync |
```
Comment (ameypv-rudder) [inline docs/sources/extract/intercom.mdx:66]: @a-rampalli could you please share the list of associated endpoints and the primary key details for the below resources? cc @nidhilashkari17

File: src/postNavList.js [hunk 2]
```diff
@@ -669,8 +669,8 @@ export const postNavList = [
   },
   {
     key: "intercom-v2",
-    title: "Intercom v2",
-    link: "/sources/extract/intercom-v2/",
+    title: "Intercom V2",
```
Comment (a-rampalli) [inline src/postNavList.js:672]: clarification: Is this V2 needed?

File: vercel.json [hunk 3]
```diff
@@ -1226,7 +1226,11 @@
...
       "source": "/cloud-extract-sources/intercom-v2/",
-      "destination": "/docs/sources/extract/intercom-v2/"
+      "destination": "/docs/sources/extract/intercom/"
+    },
+    {
+      "source": "/sources/extract/intercom-v2/",
+      "destination": "/docs/sources/extract/intercom/"
     },
     {
       "source": "/cloud-extract-sources/intercom/",
```
Comment (a-rampalli) [inline vercel.json:1236]: clarification: Can we remove this as we deprecated blendo intercom?

File: vercel.json [hunk 4]
```diff
@@ -1226,7 +1226,11 @@
     },
     {
       "source": "/cloud-extract-sources/intercom-v2/",
-      "destination": "/docs/sources/extract/intercom-v2/"
+      "destination": "/docs/sources/extract/intercom/"
+    },
+    {
+      "source": "/sources/extract/intercom-v2/",
```
Comment (a-rampalli) [inline vercel.json:1232]: If it is safe to remove below entry for blendo intercom we can remove v2 from here

File: src/postNavList.js (hunk 2 above)
Comment (ameypv-rudder) [reply src/postNavList.js:672]: this file is no longer relevant. we will eventually stop updating this.

File: vercel.json (hunk 4 above)
Comment (ameypv-rudder) [reply vercel.json:1232]: Makes sense. I will push a change re: this.

File: vercel.json (hunk 3 above)
Comment (ameypv-rudder) [reply vercel.json:1236]: @a-rampalli sure - will do!

**Review by a-rampalli — APPROVED** [review]
LGTM

File: docs/sources/extract/intercom.mdx [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+
+<div class="infoBlock">
+For more information on the schedule types, refer to the <Link to="/sources/extract/common-settings/">Common Settings</Link> guide.
+</div>
+
+### Selecting the data to import
+
+You can choose the Intercom data that you want to ingest by selecting the required resources.
+
+<img src="../../assets/cloud-extract-sources/intercomv2-5.png" alt="Selecting the data to import" />
```
Comment (ishachopra0401) [inline docs/sources/extract/intercom.mdx:54]: If we can update this image, the current one appears quite blurry

File: docs/sources/extract/intercom.mdx [hunk 6]
```diff
@@ -0,0 +1,116 @@
...
+| `segments` | Incremental | `/segments` | `id` |
+| `tags` | Full Refresh | `/tags` | `name` |
+| `teams` | Full Refresh | `/teams` | `name` |
+
+<div class="infoBlock">
+For more information on the <strong>Full Refresh</strong> and <strong>Incremental</strong> sync modes, refer to the <Link to="/sources/extract/common-settings/#sync-modes">Common Settings</Link> guide.
+</div>
+
+<div class="warningBlock">
+Any resource that supports the <strong>Incremental</strong> sync type also supports <strong>Full Refresh</strong>, by default. However, to sync your data using the <strong>Full Refresh</strong> mode, you need to <a href="mailto:support@rudderstack.com">contact the RudderStack team</a>.
```
Comment (ishachopra0401) [inline docs/sources/extract/intercom.mdx:77]:
```suggestion
Any resource that supports the <strong>Incremental</strong> sync mode also supports <strong>Full Refresh</strong>, by default. To sync your data using the <strong>Full Refresh</strong> mode in such cases, you need to <a href="mailto:support@rudderstack.com">contact the RudderStack team</a>.
```

**Review by ishachopra0401 — APPROVED** [review]

File: docs/sources/extract/intercom.mdx [hunk 7]
```diff
@@ -0,0 +1,116 @@
...
+</div>
+
+Intercom is now configured as a source. RudderStack will start ingesting data from Intercom as per your specified schedule and frequency.
+
+You can further connect this source to your data warehouse by clicking the **Add Destination** button, as shown:
+
+<img src="../../assets/cloud-extract-sources/intercomv2-6.png" alt="Configuring the source" />
```
Comment (ameypv-rudder) [inline docs/sources/extract/intercom.mdx:84]:
```suggestion
<img src="../../assets/cloud-extract-sources/add-destination.png" alt="Adding a destination" />
```

File: docs/sources/extract/intercom.mdx (hunk 5 above)
Comment (ameypv-rudder) [reply docs/sources/extract/intercom.mdx:54]: Done!

**Comment by ameypv-rudder** [comment]
Closing this PR as we are migrating to a new site. Will port the changes in a fresh PR once the new site is up.

**Comment by ameypv-rudder** [comment]
@a-rampalli please confirm about the status of this source and if we're good to release the doc.

**Comment by a-rampalli** [comment]
@ameypv-rudder There is an approval we are waiting on. Should be ready to release in a couple of days. Will update!

**Pull request closed without merging**